resources/images/controller/images.go

```diff
@@ -338,59 +363,56 @@ func (s *SoftwareImagesController) getMetaFromArchive(
 	rp := &parser.RootfsParser{}
 	aReader.Register(rp)
 
-	_, err := aReader.ReadInfo()
+	info, err := aReader.ReadInfo()
 	if err != nil {
 		return nil, errors.Wrap(err, "info error")
 	}
+	metaYocto.Info = getArtefactInfo(info)
```

Comment:
Maybe now is time to rename `*Yocto*` to `*Artifact*`. Same for `SoftwareImageMetaYoctoConstructor`.